genshi/tests/input.py

```diff
@@ -13,11 +13,11 @@
 
 import unittest
 
-from genshi.core import Attrs, Stream
-from genshi.input import XMLParser, HTMLParser, ParseError
+from genshi.core import Attrs, QName, Stream
+from genshi.input import XMLParser, HTMLParser, ParseError, ET
 from genshi.compat import StringIO, BytesIO
 from genshi.tests.test_utils import doctest_suite
-
+from xml.etree import ElementTree as et
```

Comment:
```suggestion
from xml.etree import ElementTree
```
Since we're only using this once, let's just keep the full name. One less thing for people reading the code to look up and avoids confusion with `ET`.